pylxd/models/cluster.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2016 Canonical Ltd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+from pylxd.models import _model as model
+from pylxd import managers
+
+
+class Cluster(model.Model):
+    """An LXD Cluster.
+    """
+
+    server_name = model.Attribute()
+    enabled = model.Attribute()
+    member_config = model.Attribute()
+
+    members = model.Manager()
+
+    def __init__(self, *args, **kwargs):
+        super(Cluster, self).__init__(*args, **kwargs)
+        self.members = managers.ClusterMemberManager(self.client, self)
+
+    @property
+    def api(self):
+        return self.client.api.cluster
+
+    @classmethod
+    def get(cls, client, *args):
+        """Get cluster details"""
+        print(args)
+        response = client.api.cluster.get()
+        print(response.json())
+        container = cls(client, **response.json()['metadata'])
+        return container
+
+
+class ClusterMember(model.Model):
+    """A LXD cluster member."""
+
+    name = model.Attribute(readonly=True)
```

Comment:
This needs to be `server_name` I think; I'm pretty sure it doesn't have a 'name' member

Reply:
I thought it had a name attribute because it was documented like this. I checked it against 3.7 and fixed the docu too: https://github.com/lxc/lxd/pull/5351